.circleci/config.yml

```diff
@@ -22,9 +22,11 @@ jobs:
       - image: circleci/python:3.6
     steps:
       - checkout
-      - run: pip3 install --upgrade mypy black
-      - run: black --check .
-      - run: mypy -p "brainframe.cli"
+      - run: "curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
+          | python3 - --version 1.1.0b2"
```

Comment:
Do we need the beta release of poetry?

Reply:
No, but I thought it would be smart to pin a release and I figured it should be consistent with what we're doing elsewhere.

Reply:
I think "elsewhere" is just our private repositories. It might be nice to either leave a note why, or just use the stable release

Reply:
Okay, I'll go with 1.1.0